Resuming TARDIS Simulations from HDF/Resuming TARDIS Simulations from HDF.ipynb

```diff
@@ -0,0 +1,1698 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plot_x_vs_y(data, '/simulation/plasma/t_rad', '/simulation/model/r_inner', 't_rad', 'shell index')
</pre><p>r_inner is not the same as "shell index". Instead, r_inner is the radius of the shells in centimeters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1951/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Replaced r_inner values with a series from 0 to number of shells.